### PR TITLE
Remove notify from senders message flow

### PR DESCRIPTION
**Pull request by A-Shumway42**
Added functionality to remove `US Notify` and `Notify.gov` from the users choice when they are selecting where the message/s will come from in the message send flow.

### CODE REVIEW

**Comment by ccostino** [comment]
We're closing this PR in favor of a different approach that will take place in the API instead.

File: app/main/views/send.py [hunk 1]
```diff
@@ -262,6 +264,31 @@ def set_sender(service_id, template_id):
...
+    if (
+        sender.get("sms_sender") not in ["Notify.gov", "US Notify"]
+        and not sender["is_default"]
+    ):
+        return True
+    if (
+        sender.get("sms_sender") not in ["Notify.gov", "US Notify"]
+        and sender["is_default"]
+    ):
+        return True
```
Comment (ccostino) [inline app/main/views/send.py:289]: This is a huge help seeing the breakdown in this separate method, nicely done! 🎉 

There are a couple more things, though:

- These two conditional checks are actually checking for the same thing - it doesn't matter if the sender `is_default` or not, just that they're not in the list of `["Notify.gov", "US Notify"]`! 🙂 
- We need to make sure there's a fallback return value if none of the checks actually pass, which would be `False`

We could probably simplify things and take care of the return value like so:

```suggestion
    if sender.get("sms_sender") not in ["Notify.gov", "US Notify"]:
        return True
    
    return False
```

**Review by ccostino — CHANGES_REQUESTED** [review]
Thanks, @A-Shumway42, this is nearly there! 🎉 

I had a couple of additional suggestions for the sender check to make sure we're verifying things properly; aside from that this looks great!

File: app/main/views/send.py (hunk 1 above)
Comment (A-Shumway42) [reply app/main/views/send.py]: That was completely correct! The additional logic wasn't needed. Verified it in the message send flow. Thank you for the advice and explaining that for me.

File: app/main/views/send.py [hunk 2]
```diff
@@ -262,6 +264,23 @@ def set_sender(service_id, template_id):
...
+    ]
+
+    return sender_details
+
+
+def verify_sender_options(sender):
+    if sender.get("sms_sender") not in ["Notify.gov", "US Notify"]:
+        return True
+
+    return False
```
Comment (ccostino) [inline app/main/views/send.py:281]: Whoops, it looks like one of the checks went missing!

```suggestion
    if sender.get("sms_sender") in ["Notify.gov", "US Notify"] and sender["is_default"]:
        return True

    if sender.get("sms_sender") not in ["Notify.gov", "US Notify"]:
        return True

    return False
```

**Review by ccostino — COMMENTED** [review]
One more quick change, I think!

**Review by ccostino — APPROVED** [review]
Thank you, @A-Shumway42!

**Pull request merged**